Gemfile

```diff
@@ -16,6 +16,8 @@ gem "jbuilder", "~> 2.5"
 gem "bootsnap", ">= 1.1.0", require: false
 gem "bootstrap", "~> 4.0.0"
 gem "jquery-rails"
+gem "sorcery"
```

Comment:
it's a good idea to specify the version of your gems, e.g. `gem "sorcery", "~> 0.13.0" `

Reply:
Done :)